octoprint_wled/templates/wled_navbar.jinja2

```diff
@@ -0,0 +1,12 @@
+<a class="pull-right"
+   id="wled_navbar"
+   style="float: left; font-size:20px"
+   href="javascript:void(0)"
+   data-bind="click: toggleFlashlight"
+>
+    <!-- filled bulb -->
+    <i class="fas fa-lightbulb" data-bind="visible: flashlightIsActive"></i>
```

Comment:
This one is more of a picky not-required-at-all comment, but you can use css bindings too:
```
<i class="fa-lightbulb" data-bind="css: {"fas": !flashlightIsActive(), "far": flashlightIsActive}">
```
It should be even simpler, using inline `case ? true : false` thing, but I can't remember if knockout works like that.